runners/flink/src/main/java/org/apache/beam/runners/flink/translation/wrappers/streaming/DoFnOperator.java

```diff
@@ -442,22 +443,21 @@ public void close() throws Exception {
     }
   }
 
-  private long getPushbackWatermarkHold() {
+  long getPushbackWatermarkHold() {
```

Comment:
protected?

Reply:
The reason it changed from private is to make it available to the subclass. IMO (like other similar modifier changes) it should be `protected`, because we don't want to force subclasses to the same package. `final` is orthogonal.